akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -11,13 +10,14 @@ import akka.remote.artery.SystemMessageDelivery.SystemMessageEnvelope
 import akka.serialization.{ Serialization, SerializationExtension }
 import akka.stream._
 import akka.stream.stage.{ GraphStage, GraphStageLogic, InHandler, OutHandler }
+import akka.util.OptionVal
 
 // TODO: Long UID
 class Encoder(
   uniqueLocalAddress: UniqueAddress,
   system:             ActorSystem,
   compressionTable:   LiteralCompressionTable,
-  pool:               EnvelopeBufferPool)
+  bufferPool:         EnvelopeBufferPool)
```

Comment:
`bufferBool` here and `val envelope = bufferPool.acquire()` below, a bit inconsistent